CefSharp.Test/OffScreen/OffScreenBrowserBasicFacts.cs

```diff
@@ -127,5 +130,83 @@ public async Task CanEvaluateScriptAsyncWithEncodedStringArguments()
             }
         }
 
+        class URLRequestClient : IURLRequestClient
```

Comment:
One class per file please, can move this to `CefSharp.Example` project for use in the examples.